src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraDeployMapperV1.kt

```diff
@@ -104,13 +113,13 @@ class AuroraDeployMapperV1(val applicationId: ApplicationId, val applicationFile
 
                 ),
                 resources = AuroraDeploymentConfigResources(
```

Comment:
In openShift the representation of these objects are grouped on request/limit  not on memory/cpu